### ISSUE TITLE
SmallNORB Data Creation

### ISSUE THREAD

**Issue by ssrp** [www0wwwjs1/Matrix-Capsules-EM-Tensorflow#27]
I just realized that the paper says that image samples are first normalized, then cropped, and then they add random brightness and contrast. But in your data creation code (data/smallNORB.py), you have added the random brightness and contrast before cropping.

Please add line number 164 and 165 after line number 180. It should have some effect on the accuracy I guess?

Thanks!

**Comment by kleinash** [comment]
Hi. I made this change - it doesn't do anything to the training values. I am new to tensorboard though and something odd is happening with the loss values. There is a line that goes from the start to the finish (which didn't happen before I made the change). And the loss gets to ~0.170 and then jumps to 0.3 (higher than the start). 
Also, this is taking an excessive amount of time and I have no idea how long it should take. 

![image](https://user-images.githubusercontent.com/2494123/39267104-d9606fee-4899-11e8-9bc2-25ce427ca2e3.png)
